backend/loi-service/src/main/java/org/egov/works/web/models/APIInfo.java

```diff
@@ -0,0 +1,41 @@
+package org.egov.works.web.models;
+
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.springframework.validation.annotation.Validated;
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Builder;
+
+/**
+ * Info of the API being called
+ */
+@ApiModel(description = "Info of the API being called")
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-08-04T15:05:28.525+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class APIInfo   {
+        @JsonProperty("id")

```

Comment:
Use from `services-common` or `digit-models` instead of defining these classes in this project.

Reply:
I think APIInfo is not present in  digit-models or services-common it was also part of this PR #11

Reply:
Still, remove this. As we are using RequestInfo from `services-common` and it is not using APIInfo, we are not using this class anywhere.

Reply:
done